_includes/cloud-query/cloud-query-execute.md

```diff
@@ -2,5 +2,5 @@ There are two methods for executing a query in the UI:
 
 |Method | Instructions  |
 | --- | ----------- |
-| Click to run | Click the button in the top left labeled **RUN** and select your database.  |
+| Click to run | Click the button in the top left labeled **RUN**  |
 | Hot keys | Place your cursor anywhere within the query you'd like to run in the text editor and hit a key combination of `“Ctrl + Enter"` or `Cmd + Enter"` (depending on your OS and preference) |
```

Comment:
This is another of those lots of words that can be removed situations with some additional comments:

* The user can't place the cursor anywhere because "placing" is a physical action. The user CAN click the query field 
* If there's more than one possible combination, these should be made clearer

So in this case we would go something like this:

```
There are three methods to run a query:

| Medium |  Steps |
|---|---|
| FeatureBase Cloud UI | * Click **Run** |
| MacOS | * Click in the query<br/>* Click `Cmd` + `Enter` | 
| Linux<br/>Windows | * Click in the query<br/>* Click `Cmd + Enter |
```